src/desktop/apps/auction/components/Layout.js

```diff
@@ -29,12 +29,9 @@ function Layout(props) {
 
   const b = block("auction-Layout")
 
-  const Modal =
-    modal === "ConfirmRegistration" ? ConfirmRegistrationModal : null
-
   return (
     <div className={b()}>
-      {Modal && <Modal />}
+      {modal && <ConfirmRegistrationModal />}
```

Comment:
We should start migrating some of the raw JavaScript files to TypeScript, so it'll be clear what "modal" refers to.

Reply:
yes, @dleve123 and i have already talked about renaming this variable. It's not good!